eclair-core/src/main/scala/fr/acinq/eclair/wire/ChannelCodecs.scala

```diff
@@ -52,7 +52,7 @@ object ChannelCodecs extends Logging {
 
   val channelVersionCodec: Codec[ChannelVersion] = discriminatorWithDefault[ChannelVersion](
     discriminator = discriminated[ChannelVersion].by(byte)
-      .typecase(0x01, provide(ChannelVersion.STANDARD))
+      .typecase(0x01, bits(4 * 8).as[ChannelVersion])
```

Comment:
Why hardcoding the bit values instead of using `ChannelVersion.STANDARD.bits` here?